README.md

```diff
@@ -3,9 +3,13 @@
 just execute
 ```
 npm install
-node bin/index.js --token=YOUR_GITHUB_TOKEN --source=PAYLOAD_URL_TO_BE_CHANGED --target=NEW_PAYLOAD_URL --file=THE_FILE_PATH_CONTAINING_REPOSITORY_LIST
+node bin/index.js --token=YOUR_GITHUB_TOKEN --source=WEBHOOK_PAYLOAD_URL --file=THE_FILE_PATH_CONTAINING_REPOSITORY_LIST [--url=NEW_PAYLOAD_URL --content_type=NEW_CONTENT_TYPE --secret=NEW_SECRET --insecure_ssl=NEW_INSECURE_SSL --address=NEW_ADDRESS --room=NEW_ROOM]
 ```
 
+All those parameters in `[` and `]` are optional, this means that only those that should be changed must be provided. Note that if none of them is provided the webhook configuration won't be changed at all.
```

Comment:
In case this is a fact, url arg should be the same as source (I mean it should be overwritten), since `config.url` is required, no need to define url from CLI in case you don't mean to update URL.
Please, clarify `source` arg is just for webhook filtering

Reply:
yeah that's correct as we will inherith the configuration from the filtered webhook, therefore no need to set the `url` arg if we don't want to change it.

Added clarification here https://github.com/Ginxo/github-api-massive/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R11